### ISSUE TITLE
Getting "Failed to load plugin dashboards" error

### ISSUE THREAD

**Issue by NRS229** [trey-wallis/obsidian-dashboards#28]
### Describe the bug

I'm trying to activate the plugin in the Community Plugins page

### Steps to reproduce

Activate plugin

### Expected behavior

Plugin activation

### Obsidian debug info

SYSTEM INFO:
	Obsidian version: v1.4.14
	Installer version: v1.4.12
	Operating system: Darwin Kernel Version 22.6.0: Wed Jul  5 22:21:56 PDT 2023; root:xnu-8796.141.3~6/RELEASE_X86_64 22.6.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: AnuPpuccin v1.4.2
	Snippets enabled: 1
	Restricted mode: off
	Plugins installed: 16
	Plugins enabled: 15
		1: Advanced Tables v0.18.1
		2: Tasks v3.6.0
		3: AidenLx's Folder Note v0.16.5
		4: Folder Note Core v1.3.5
		5: AidenLx's Folder Note - folderv component v1.0.0
		6: Waypoint v1.4.0
		7: Chronology v1.1.4
		8: Kanban v1.5.3
		9: Heatmap Calendar v0.6.0
		10: Homepage v3.3.3
		11: Reminder v1.1.15
		12: Hider v1.2.5
		13: Editor Syntax Highlight v0.1.3
		14: Dataview v0.5.55
		15: MetaEdit v1.8.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by trey-wallis** [comment]
Hi @NRS229. DataLoom was formerly named Dashboards and Notion-Like Tables. Please delete the `notion-like-tables` folder. It be found in `<your-vault>/.obsidian/plugins/notion-like-tables`. Then please download `DataLoom`

**Comment by NRS229** [comment]
Does the new Dataloom plugin has the "Create a grid layout" function?

**Comment by NRS229** [comment]
Does the new Dataloom plugin include the "Create a grid layout" function?

**Comment by trey-wallis** [comment]
No, it doesn't. DataLoom has a tabular layout and it is used to manage and
edit tabular data. Dashboards as the name indicates was used to create a
grid in which you could embed different links.

I am no longer working on Dashboards because I feel there's enough support
for dashboard-like design in regular markdown files without using a custom
view.

**Comment by trey-wallis** [comment]
@NRS229 I would recommend checking out the Dashboards++ method (https://github.com/TfTHacker/DashboardPlusPlus) and also the Home Page plugin (https://github.com/mirnovov/obsidian-homepage)
